packages/@aws-cdk/aws-efs/lib/access-point.ts

```diff
@@ -0,0 +1,171 @@
+import { Construct, IResource, Resource, Stack } from '@aws-cdk/core';
+import { IFileSystem } from './efs-file-system';
+import { CfnAccessPoint } from './efs.generated';
+
+/**
+ * Represents an EFS AccessPoint
+ */
+export interface IAccessPoint extends IResource {
+  /**
+   * The ID of the AccessPoint
+   *
+   * @attribute
+   */
+  readonly accessPointId: string;
+
+  /**
+   * The ARN of the AccessPoint
+   *
+   * @attribute
+   */
+  readonly accessPointArn: string;
+}
+
+/**
+ * Represents the CreateionInfo
+ */
+export interface CreationInfo {
+  /**
+   * Specifies the POSIX user ID to apply to the RootDirectory. Accepts values from 0 to 2^32 (4294967295).
+   */
+  readonly ownerUid: string;
+
+  /**
+   * Specifies the POSIX group ID to apply to the RootDirectory. Accepts values from 0 to 2^32 (4294967295).
+   */
+  readonly ownerGid: string;
+
+  /**
+   * Specifies the POSIX permissions to apply to the RootDirectory, in the format of an octal number representing
+   * the file's mode bits.
+   */
+  readonly permissions: string;
+}
+
+/**
+ * Represents the PosixUser
+ */
+export interface PosixUser {
+  /**
+   * The POSIX user ID used for all file system operations using this access point.
+   */
+  readonly uid: string;
+
+  /**
+   * The POSIX group ID used for all file system operations using this access point.
+   */
+  readonly gid: string;
+
+  /**
+   * Secondary POSIX group IDs used for all file system operations using this access point.
+   *
+   * @default - None
+   */
+  readonly secondaryGids?: string[];
+}
+
+/**
+ * Properties for the AccessPoint
+ */
+export interface AccessPointProps {
+  /**
+   * The efs filesystem
+   */
+  readonly fileSystem: IFileSystem;
+
+  /**
+   * Specifies the POSIX IDs and permissions to apply to the access point's RootDirectory. If the
+   * `RootDirectory` > `Path` specified does not exist, EFS creates the root directory using the `CreationInfo`
+   *  settings when a client connects to an access point.
+   *
+   * @default - None
+   */
+  readonly creationInfo?: CreationInfo;
+
+  /**
+   * Specifies the path on the EFS file system to expose as the root directory to NFS clients using the access point
+   * to access the EFS file system
+   *
+   * @default - root(/) directory of the efs filesystem
+   */
+  readonly path?: string;
+
+  /**
+   * The full POSIX identity, including the user ID, group ID, and any secondary group IDs, on the access point
+   *  that is used for all file system operations performed by NFS clients using the access point.
+   *
+   * @default - None
```

Comment:
What does 'None' mean here? What user id does EFS use when not specified?

Reply:
Then the user identity enforcement will not enabled. I've added additional note in the docstring. Hope it looks much better.